app/src/RoomClient.js

```diff
@@ -1452,9 +1460,20 @@ export default class RoomClient
 
 				store.dispatch(settingsActions.setSelectedAudioDevice(trackDeviceId));
 
+				const networkPriority =
+					window.config.networkPriorities.audio ?
```

Comment:
@misi we can use this syntax `window.config.networkPriorities?.audio ?` to avoid errors with old configurations where networkPriorities is missing.